targets/targets.json

```diff
@@ -7965,12 +8019,21 @@
             "network-default-interface-type": "WIFI"
         }
     },
+    "CYW943012P6EVB_01_M0": {
+        "inherits": ["MCU_PSOC6_M0"],
+        "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
+        "extra_labels_add": ["CY8C62XX", "CYW943012P6EVB_01"],
+        "macros_add": ["CY8C6247BZI_D54", "PSOC6_DYNSRM_DISABLE=1"],
+        "post_binary_hook": {
+            "function": "PSOC6Code.complete"
+        }
+    },
     "CYW943012P6EVB_01": {
         "inherits": ["MCU_PSOC6_M4"],
         "features": ["BLE"],
         "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
```

Comment:
No need to specify supported_toolchains here as the parent MCU_PSOC6 already defines the same toolchain set as here.

Reply:
done